### PR TITLE
Fixes due to updated nam.inna.is specifications

### PR DESCRIPTION
**Pull request by HermannBjorgvin**
Fixes for Inna updates to API naming scheme, handling of cookies, key exceptions in schedule.json that didn't exist before. Added a file dump for the schedule JSON info as well.

### CODE REVIEW

File: InnaCalendar.py [hunk 1]
```diff
@@ -58,15 +62,26 @@
 
 print("Fetching Schedule...")
 
-schedule = requests.get("https://nam.inna.is/inna11/api/Timetable/GetTimetable", params=schedulePayload, cookies=newCookie) # Get the user schedule with our payload
+schedule = requests.get("https://nam.inna.is/api/Timetable/GetTimetable", params=schedulePayload, cookies=newCookie) # Get the user schedule with our payload
 js = schedule.json() # Store our JSON Object
 
+print("Saving JSON for debugging...")
```
Comment (Glitch-is) [inline InnaCalendar.py]: Get rid of debugging

File: InnaCalendar.py [hunk 2]
```diff
@@ -58,15 +62,26 @@
...
+  json.dump(js, f, ensure_ascii=False)
+
 print("Starting to build Schedule...")
 
 cal = Calendar() # Initiate a new Calander Object
 
 for i in js: # Loop through each class individually
-    title = i["titleShort"] + " " + i["classroom"] + " " + i["teacher"] # Title of the event
+    # putting teacher in a variable to catch exceptions. Sometimes no teacher is registered and we get keyerror
+    teacher = ""
```
Comment (Glitch-is) [inline InnaCalendar.py]: This can be simplified to

```python
teacher = " " + i["teacher"] if 'teacher' in i else ""
```

**Review by Glitch-is — CHANGES_REQUESTED** [review]
Looks fine, I'd just like some minor changes

**Review by Glitch-is — APPROVED** [review]

**Comment by HermannBjorgvin** [comment]
Added commit with fixes

**Pull request merged**